prr/services/providers/google/chat.py

```diff
@@ -0,0 +1,104 @@
+from google.cloud import aiplatform
+from vertexai.preview.language_models import ChatModel, InputOutputTextPair
+
+from prr.config import load_config
+from prr.request import ServiceRequest
+from prr.response import ServiceResponse
+
+config = load_config()
+
+aiplatform.init(
+    # your Google Cloud Project ID or number
+    # environment default used is not set
+    project=config.get("GOOGLE_PROJECT"),
```

Comment:
Let's add these to .env.example so people know what to do without reading the source.